### ISSUE TITLE
Simultaneous Mono/CSpect/mono-sgen64 spawn

### ISSUE THREAD

**Issue by NealeTools** [remy/vscode-nextbasic#58]
On prior versions, launching CSpect used kill any current mono instance before launching.
On new plugin, this doens't happen and I can inadvertently get a few CSpect running in parallel, slowing each other down, and eating huge laptop resources.
Secondly, even when quitting them, sometimes they leave behind this "mono-sgen64" process (two of them) that are massive resource hogs (200% CPU each). On worst case, even when closing laptop to sleep, it left these processing running and fans blowing. ie. They prevented sleep! Hoping this is an easy fix :-)

MacOS 10.15.7 MBPro i7 13 inch.

<img width="662" alt="image" src="https://github.com/remy/vscode-nextbasic/assets/19664927/b485f379-78b1-457f-b2cf-2d7ede403468">

**Comment by remy** [comment]
I've not really touched anything regarding how cspect is launched....

Is it safe to assume you've not changed version of cspect and your Macos hasn't been upgraded in-between?

**Comment by NealeTools** [comment]
No changes to macOS or CSpect.
I can reproduce with every version from v1.11.5+
Earlier versions (v1.11.4 and earlier) appear to successfully kill the active mono process.

**Comment by NealeTools** [comment]
This is still an issue in v1.11.8, sorry!

**Comment by remy** [comment]
Yeah, this issue is outside of the txt2bas, equally, there was no code
change around this logic, my gut says it's either vscode or the OS that's
the issue with it launching...

Though, it might be easy to confirm by installing a previous version of the
extension to see if the issue repeats.

**Comment by NealeTools** [comment]
Thanks Remy: See comment on Apr 29. The issue started in extension version 1.11.5 onwards, so should hopefully be a simple fix to see what changed between version 1.11.4 and 1.11.5. (Please :-) - the multiple spawning makes it pretty unusable at the moment. I regularly (also now in v.1.11.8) end up with a phantom task that spins up the laptop fans.

**Comment by remy** [comment]
hmm, just spotted something. just putting kids down then will push a new version, might be it - not sure why it would be, but it's worth trying

**Comment by remy** [comment]
Well… I tried to upload a pre-release, but for some reason, it landed as "real" release. So that one word change is now in 1.11.9 - if you want to give it a try

**Comment by NealeTools** [comment]
Perfect! This kills it every time now 👍
